recipes/robotframework/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "robotframework" %}
+{% set version = "3.0.2" %}
+{% set sha256 = "2018f233568227f9af8c366f93e243a157b742f3baf5b649ecc9750ac0ee1ff7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: robotframework-3.0.2.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Could this be `noarch: python`?

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
Absolutely! Good point!